share/cyclus-flat.rng.in

```diff
@@ -28,6 +28,12 @@ datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
         <element name="explicit_inventory_compact"> <data type="boolean"/> </element>
       </optional>
       <optional>
+          <element name="eps"><data type="double"/></element>
```

Comment:
I wonder whether these should have slightly more explicit names in the xml?  Since this is user-facing, it's important to get it right.  Some options:
* epsilongeneric, epsilonresource
* epsilon_generic, epsilon_resource
* tolerance_generic, tolerance_resource